### ISSUE TITLE
Development info on issue cards

### ISSUE THREAD

**Issue by johvik** [MeetMe/jira-gerrit-plugin#65]
Jira Software 8.1.x added support for development info on issue cards:
![image](https://user-images.githubusercontent.com/2324954/58456612-eefeea00-8124-11e9-82e1-49be5de5df3d.png)

Would it be possible to make this plugin a development status provider (or similar)?

**Comment by jhansche** [comment]
The problem with this is that the "Development" tools are provided as a DVCS plugin, which is not (or was not at the time, that may have changed?) publicly documented or accessible.

If Atlassian has made the Development panels accessible to plugins in a way that we can contribute statuses, commits, and reviews to the DVCS plugin for display in the Development panels, then I agree it would be great to integrate with that.

**Comment by gonchik** [comment]
Hi! 
@jhansche  do you mean that one ? 
https://bitbucket.org/atlassian/jira-dvcs-connector/src/master/

Just interesting gitlab  EE has the same integration:
https://docs.gitlab.com/ee/integration/jira_development_panel.html
but CE does not have
https://gitlab.com/gitlab-org/gitlab-ce/issues/46117

Hope it helps.

**Comment by jhansche** [comment]
Yes, that is all correct. And as you can see from the gitlab links you provided, there is no official support from Atlassian for this.

I suspect that as the Development Status plugin matures, they may start exposing more plugin extension points here. But until then, anything else we do to enable integration with it will be a completely unsupported hack.

Even the atlassian PR and gitlab issue explain as much. They "emulate" a github installation in order to allow gitlab repos to appear in the DVCS/Development panels, as if they were github projects. And the other option is a man-in-the-middle proxy that emulates github and then interfaces with gitlab.

Neither of those options (for _full_ Development plugin support) I'm willing to entertain at this time.

If there are lighter-weight options for getting those bits of information into other locations, that would be a reasonable short term solution.  I do not believe that to be the case.

Given that the Development plugin has changed drastically over the last several years, and still has no plugin extension support, I suspect that this implementation will not happen any time soon. I'm leaving it open specifically because I agree, and I **would like** to support the Development panels. I just don't want to do it with a fragile hack that may well break after every minor version of Jira.

**Comment by tikeyknax** [comment]
Are there any news about this issue? The last comment 5 years ago says that there might changes, so I'm wondering if there are any news.
